### PR TITLE
feat: added function to compare phones

### PR DESCRIPTION
**Pull request by caroltrindade**
### Description

I added a function to compare phones and another function to remove the first digit from the phone (I use it in the first function xD)

### CODE REVIEW

File: __tests__/phone/validators.test.ts [hunk 1]
```diff
@@ -14,3 +14,25 @@ describe('isBrazilianPhone', () => {
...
+  const samePhones = [
+    { phone1: '47996919816', phone2: '47996919816' },
+    { phone1: '4796919816', phone2: '4796919816' },
+    { phone1: '47996919816', phone2: '4796919816' }
+  ]
+
+  const differentPhones = [
+    { phone1: '47996919816', phone2: '47986919816' },
+    { phone1: '4796919816', phone2: '4786919816' },
+    { phone1: '55996919816', phone2: '47996919816' }
```
Comment (alanneves) [inline __tests__/phone/validators.test.ts:28]: in my opinion it should be the same phone, is not?

**Review by alanneves — CHANGES_REQUESTED** [review]

File: __tests__/phone/validators.test.ts (hunk 1 above)
Comment (caroltrindade) [reply __tests__/phone/validators.test.ts:28]: No because the DDD

Comment (alanneves) [reply __tests__/phone/validators.test.ts:28]: can you add a test in the valid same phones, lie:

5547996919817
47996919817

because they should be true

Comment (caroltrindade) [reply __tests__/phone/validators.test.ts:28]: Oh, I forgot the DDI, thank youu

**Review by alanneves — APPROVED** [review]
LGTM

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by alanneves — APPROVED** [review]
LGTM

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by alanneves — APPROVED** [review]
LG|TM

**Pull request merged**